### DISCUSSION TITLE
How to access htmlinputelement wrapped by QInput in v2

### DISCUSSION THREAD

**Discussion by mdafzal** [quasarframework/quasar#10221] [General - Components / Directives / etc] [answered]
`<q-input ref="myinput" />`

To access html input element wrapped by qinput component, this.$refs['myinput'].$refs['input'] works in Quasar v1.

But it is not working in Quasar v2 anymore

**Comment by hawkeye64** [comment]
I think what you mean to say is "Vue 3 does not allow chained refs like Vue 2 did"...

**Reply by mdafzal** [reply]
I need HTMLInputElement to make it work with Google maps. Is there a way to get inner HtmlInputElement of QInput component in Quasar v2 ?

**Comment by mdafzal** [comment, marked as answer]
To access native element, used getNativeElement method provided by Quasar API like this: `this.$refs['myinput'].getNativeElement()` and it worked

https://quasar.dev/vue-components/input#qinput-api
